docs/exclusive-content.md

```diff
@@ -212,3 +204,41 @@ If you see the source files instead of the example, click **View App** in the bo
     style={{ height: '100%', width: '100%', border: '0' }}>
   </iframe>
 </div>
+
+## Exclusive content with payment verification
+
+The above examples only hide content client side which could be spoofed by a clever user.
+Since the introduction of STREAM receipts it is possible to verify payments using a STREAM receipt verifier.
+
+The Exclusive Content Generator allows users to generate an encrypted piece of content that can be embedded on the web page.
+
+:::info
+Want to try the Exclusive Content Generator right away? Check it out [here](/exclusive-content).
+:::
+
+### How does it work?
+
+The Exclusive Content Generator is a Cloudflare worker that derives an encryption key from the user's payment pointer and a master key and returns it to the Creator's client. The client then encrypts the exclusive content and the verifier endpoint and displays all the information required for decryption in a <code>div</code> tag. It also provides a JavaScipt scipt to be embedded on the Creator's web page.
+
+[![](https://mermaid.ink/img/eyJjb2RlIjoic2VxdWVuY2VEaWFncmFtXG5hdXRvbnVtYmVyXG5DcmVhdG9yIC0-PiBDcmVhdG9yJ3MgQ2xpZW50OiBwYXltZW50IHBvaW50ZXIsIGNvbnRlbnQsIHZlcmlmaWVyIGVuZHBvaW50IChvcHRpb25hbClcbkNyZWF0b3IncyBDbGllbnQgLT4-IEV4Y2x1c2l2ZSBDb250ZW50IEdlbmVyYXRvcjogcGF5bWVudCBwb2ludGVyXG5FeGNsdXNpdmUgQ29udGVudCBHZW5lcmF0b3IgLT4-IEV4Y2x1c2l2ZSBDb250ZW50IEdlbmVyYXRvcjogZGVyaXZlcyBlbmNyeXB0aW9uIGtleSBmcm9tIHBheW1lbnQgcG9pbnRlclxuRXhjbHVzaXZlIENvbnRlbnQgR2VuZXJhdG9yIC0-PiBDcmVhdG9yJ3MgQ2xpZW50OiBlbmNyeXB0aW9uIGtleVxuQ3JlYXRvcidzIENsaWVudCAtPj4gQ3JlYXRvcidzIENsaWVudDogZW5jcnlwdHMgY29udGVudCBhbmQgdmVyaWZpZXIgZW5kcG9pbnQsIGdlbmVyYXRlcyBkaXYgdGFnXG5DcmVhdG9yJ3MgQ2xpZW50IC0-PiBDcmVhdG9yOiBkaXYgYW5kIHNjcmlwdCB0YWdzIiwibWVybWFpZCI6eyJ0aGVtZSI6ImRlZmF1bHQifSwidXBkYXRlRWRpdG9yIjpmYWxzZX0)](https://mermaid-js.github.io/mermaid-live-editor/#/edit/eyJjb2RlIjoic2VxdWVuY2VEaWFncmFtXG5hdXRvbnVtYmVyXG5DcmVhdG9yIC0-PiBDcmVhdG9yJ3MgQ2xpZW50OiBwYXltZW50IHBvaW50ZXIsIGNvbnRlbnQsIHZlcmlmaWVyIGVuZHBvaW50IChvcHRpb25hbClcbkNyZWF0b3IncyBDbGllbnQgLT4-IEV4Y2x1c2l2ZSBDb250ZW50IEdlbmVyYXRvcjogcGF5bWVudCBwb2ludGVyXG5FeGNsdXNpdmUgQ29udGVudCBHZW5lcmF0b3IgLT4-IEV4Y2x1c2l2ZSBDb250ZW50IEdlbmVyYXRvcjogZGVyaXZlcyBlbmNyeXB0aW9uIGtleSBmcm9tIHBheW1lbnQgcG9pbnRlclxuRXhjbHVzaXZlIENvbnRlbnQgR2VuZXJhdG9yIC0-PiBDcmVhdG9yJ3MgQ2xpZW50OiBlbmNyeXB0aW9uIGtleVxuQ3JlYXRvcidzIENsaWVudCAtPj4gQ3JlYXRvcidzIENsaWVudDogZW5jcnlwdHMgY29udGVudCBhbmQgdmVyaWZpZXIgZW5kcG9pbnQsIGdlbmVyYXRlcyBkaXYgdGFnXG5DcmVhdG9yJ3MgQ2xpZW50IC0-PiBDcmVhdG9yOiBkaXYgYW5kIHNjcmlwdCB0YWdzIiwibWVybWFpZCI6eyJ0aGVtZSI6ImRlZmF1bHQifSwidXBkYXRlRWRpdG9yIjpmYWxzZX0)
```

Comment:
I'll change that to include the svg in the assets as soon as we are happy with how it looks.